respect-lib-shared/src/commonMain/kotlin/world/respect/shared/viewmodel/person/manageaccount/ManageAccountViewModel.kt

```diff
@@ -44,6 +53,12 @@ data class ManageAccountUiState(
     val personPassword: DataLoadState<PersonPassword> = DataLoadingState(),
     val errorText: UiText? = null,
     val selectedAccount: RespectSessionAndPerson? = null,
+    val isStudent: Boolean = false,
+    val qrBadge: DataLoadState<PersonBadge> = DataLoadingState(),
+    val showBottomSheet: Boolean = false,
+    val badgeNumber: String? = null,
```

Comment:
badgeNumber **should not** be a separate property. The Badge number is already part of the qrBadge. Duplication is almost always bad: if the two are different, which one should be shown?

if needed you can add a property with no backing variable to make it quicker/shorter for commonly accessed things e.g.

```
val badgeNumber: String?
   get() = qrBadge.dataOrNull().badgeNumber
```